pkg/common/common.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	httpTimeout        = 5 * time.Minute
+	httpTimeout        = 30 * time.Second
```

Comment:
Does it make sense to relate this to the maxRetries and retryDelay?

Reply:
Yeah, need to read more about this, looking on the comment, it seems that it can interrupt the body read if the timeout is reached, and in the case of huge blobs, I need to read and test it more in depth